scripts/train.py

```diff
@@ -0,0 +1,41 @@
+import os
+import click
+import pickle
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from sklearn.neural_network import MLPClassifier
+from pkspace.utils.loaders import PKSpaceLoader, PKLotLoader
+from pkspace.utils import trainer
+
+
+@click.command()
+@click.option('--PKSpace', 'dataset_mode', flag_value='PKSpace',
+              default=True)
+@click.option('--PKLot', 'dataset_mode', flag_value='PKLot')
+@click.option('--MLP', 'method', flag_value='MLP',
+              default=True, help='Method to be used for prediction')
+@click.option('--dataset_dir', required=True,
+              help='Directory of dataset for model to be trained on')
+@click.option('--output', default=None,
+              help='Name of output file for trained model')
```

Comment:
That is fine. We can still have `output = os.path.join(dataset_dir, 'out.pkl')`. I'm sorry if I was unclear. I just wanted to add the default name of the file to the argument so we wouldn't have to check if it is `None`. But now I agree it is for consideration if the output argument should specify the whole path to the output file. I think it is up to you what to do with it. Now I think this is better because it adds flexibility to the script and user.